src/main/webapp/app/admin/report-management/report-management-log.controller.js

```diff
@@ -94,10 +95,26 @@
         function viewJiraTicket(log) {
             $window.open(log.viewTicket, '_blank');
         }
-        function setTicketCreation(log, enableTicketCreation) {
-            if (!log.isTicketCreated) {
-                log.enableTicketCreation = enableTicketCreation;
+       
+        function enableTicketCreation(log) {
+            if (!log.isTicketCreated && log.thresholdMet) {
+                log.enableTicketCreation = !log.enableTicketCreation;
+                schedulerService.disableTicketCreation(log.schedulerTaskMetaId).then(
+                    function (response) {
+
+                    },
+                    function (error) {
+                        $rootScope.showErrorSingleToast({
+                            text: 'Error while disable ticket creation',
+                            title: "Error"
+                        });
+                    });
+            }
+        }
+        function getLabelClass(log) {
+            if (log.isTicketCreated == true || log.thresholdMet == false || log.taskStatus != "success") {
```

Comment:
it can be like this
if(log.isTicketCreated || log.thresholdMet || log.taskStatus !== "success")
try to use === instead of ==